### PR TITLE
community: adding langchain-predictionguard partner package documentation

### PR DESCRIPTION
**Pull request by jmansdorfer**
- *[x] **PR title**: "community: adding langchain-predictionguard partner package documentation"

- *[x] **PR message**:
    - **Description:** This PR adds documentation for the langchain-predictionguard package to main langchain repo, along with deprecating current Prediction Guard LLMs package. The LLMs package was previously broken, so I also updated it one final time to allow it to continue working from this point onward. . This enables users to chat with LLMs through the Prediction Guard ecosystem.
    - **Package Links**: 
        -  [PyPI](https://pypi.org/project/langchain-predictionguard/)
        -  [Github Repo](https://www.github.com/predictionguard/langchain-predictionguard)
    - **Issue:** None
    - **Dependencies:** None
    - **Twitter handle:** [@predictionguard](https://x.com/predictionguard)

- *[x] **Add tests and docs**: All docs have been added for the partner package, and the current LLMs package test was updated to reflect changes.

- *[x] **Lint and test**: Linting tests are all passing.

### CODE REVIEW

**Review by efriis — COMMENTED** [review]
hey there! could you also register the repo in `libs/packages.yml`?

**Comment by jmansdorfer** [comment]
@efriis should be added now. Thanks for pointing that out!

File: docs/docs/integrations/providers/predictionguard.mdx [hunk 1]
```diff
@@ -1,102 +1,78 @@
-# Prediction Guard
+# langchain-predictionguard
 
 This page covers how to use the Prediction Guard ecosystem within LangChain.
 It is broken into two parts: installation and setup, and then references to specific Prediction Guard wrappers.
```
Comment (ccurme) [inline docs/docs/integrations/providers/predictionguard.mdx:4]:
```suggestion
# Prediction Guard

This page covers how to use the Prediction Guard ecosystem within LangChain.
It is broken into two parts: installation and setup, and then references to specific Prediction Guard wrappers.

This integration is maintained in the [langchain-predictionguard](https://github.com/predictionguard/langchain-predictionguard)
package.
```

nitpick but can we retitle to keep this consistent with other provider pages?

**Review by ccurme — APPROVED** [review]

**Comment by jmansdorfer** [comment]
@ccurme For sure! Should be fixed now.

**Review by ccurme — APPROVED** [review]

**Pull request merged**